Makefile

```diff
@@ -305,27 +305,31 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 	# Install cert manager.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/cert-manager.yaml
-	# Wait for cert-manager pods to be created
-	sleep 20
-	# Wait for cert-manager pods to be ready.
+		apply -f examples/_out/cert-manager.yaml
+	# Wait for webhook servers to be ready to take requests
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		wait --for=condition=Ready --namespace=cert-manager --timeout=15m pods --all
-	# Wait for webhook servers to be ready to take requests
-	sleep 10
+		wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
 	# Apply provider-components.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/provider-components.yaml
+		apply -f examples/_out/provider-components.yaml
+	# Wait for CAPI pod
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		wait --for=condition=Ready --timeout=5m -n capi-system pod -l control-plane=cluster-api-controller-manager
+    # Wait for CAPA pod
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		wait --for=condition=Ready --timeout=5m -n capa-system pod -l control-plane=capa-controller-manager
 	# Create Cluster.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/cluster.yaml
+		apply -f examples/_out/cluster.yaml
```

Comment:
Why apply here? I'd worry that we could potentially mutate something existing vs creating from scratch here.

Reply:
I was manually rerunning selective commands from here, and you can't do that with create.

As the code stands, this is operating against a brand new, clean kind cluster. There is nothing to overwrite. Going forward, if we proceed with requiring a valid kubeconfig (and therefore a cluster) instead of having this make target create one, I think apply is still good. This target is really "install capi + capa into a management cluster and then create a workload cluster".

Reply:
My worry with apply is that the state may not necessarily be what we expect it to be (the equivalent of a new clean deployment of capi + capa components), or that we could introduce changes that would break deployment from a clean cluster inadvertently to the manifests (requiring some new additional pre-requisite in the management cluster) that we previously didn't. The use of apply could potentially mask these until release time (or a new developer tries to replicate).

Reply:
Reverted